decidim/decidim
# Fix missing icons in meetings

#### :tophat: What? Why?
While reviewing #13570, i noticed the only place in decidim where the `decidim_url_escape` helper is being used is in the meeting minutes page. Upon setting video url and audio url, i noticed there are 2 exceptions regarding missing icons. 
This PR registers the missing icons 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #13570

#### Testing
1. Visit the close meeting interface from admin 
2. Set audio and video urls and tick check for "_Is visible_"
3. Close the meeting 
4. Visit the public website, see the page is having an error 
5. Apply patch & restart server
6. refresh & see there is no error

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![image](https://github.com/user-attachments/assets/8bcd74b8-fdc7-4912-8fe9-a2e700b5db20)

:hearts: Thank you!
